doc/source/whatsnew/v0.18.0.txt

```diff
@@ -870,6 +870,7 @@ In prior versions, new columns assignments in an ``eval`` expression resulted
 in an inplace change to the ``DataFrame``. (:issue:`9297`)
 
 .. ipython:: python
+   :okwarning:
 
    df = pd.DataFrame({'a': np.linspace(0, 10, 5), 'b': range(5)})
    df.eval('c = a + b')
```

Comment:
This is about previous behaviour, so maybe use  code-block? (the warning should not be there, as this is introduced in 0.18)